bitnami/fluentd/values-production.yaml

```diff
@@ -39,6 +39,15 @@ clusterDomain: cluster.local
 
 forwarder:
   enabled: true
+
+  ## K8s Security Context for forwarder pods
+  ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  ##
+  securityContext:
+    enabled: true
+    runAsUser: 1001
+    fsGroup: 1001
+
```

Comment:
This is different than in the `values.yaml` file, if that is intended please document the differences on the README.